instaclustr/resource_cluster.go

```diff
@@ -69,12 +69,26 @@ func resourceCluster() *schema.Resource {
 				Optional: true,
 			},
 
+			"public_contact_addresses": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Optional: true,
```

Comment:
try removing the optional, just go with `Computed: true,`

Reply:
yeah, I have tried that...... many a time... this is what I tohught also... here we go again. 👍